src/components/common/AuthenticationAlert.js

```diff
@@ -20,4 +20,9 @@ class AuthenticationAlert extends React.PureComponent {
   }
 }
 
-export default AuthenticationAlert;
+export class VPNAlert extends AuthenticationAlert {
```

Comment:
This file is named `AuthenticatoinAlert.js`, after the first component. Either the file should be renamed, or `VPNAlert` should be moved to another file.

Reply:
I'll move this to another file. Because I can't think of a good name that encapsulates both. VPN and Authentication are related but there's no good word to contain both other than "ACL" or "Access Woes"

Reply:
Maybe something like `alerts.js` would work? Though perhaps *too* broad. I'm fine with either.